addons/website_sale/models/res_config_settings.py

```diff
@@ -116,6 +127,16 @@ def _inverse_account_on_checkout(self):
             else:
                 record.website_id.auth_signup_uninvited = 'b2b'
 
+    @api.depends('website_id.ecommerce_access')
+    def _compute_ecommerce_access(self):
+        for record in self:
+            record.ecommerce_access = record.website_id.ecommerce_access or 'everyone'
```

Comment:
Since `website_id.ecommerce_access` is by default 'everyone', it is not needed.
```suggestion
            record.ecommerce_access = record.website_id.ecommerce_access
```